app/orders/fiori-service.cds

```diff
@@ -24,7 +24,32 @@ annotate AdminService.OrderItems with {
 	);
 }
 
+annotate AdminService.Orders with {
+	shippingAddress @(
+		Common: {
+			FieldControl: #Mandatory,
+			ValueList: {
+				CollectionPath: 'Addresses',
+				Label: 'Addresses',
+				SearchSupported: 'true',
+				Parameters: [
+					{ $Type: 'Common.ValueListParameterOut', LocalDataProperty: 'shippingAddress_AddressID', ValueListProperty: 'AddressID'},
+					{ $Type: 'Common.ValueListParameterOut', LocalDataProperty: 'shippingAddress_BusinessPartner', ValueListProperty: 'BusinessPartner'},
+					{ $Type: 'Common.ValueListParameterDisplayOnly', ValueListProperty: 'PostalCode'},
+					{ $Type: 'Common.ValueListParameterDisplayOnly', ValueListProperty: 'CityName'},
+					{ $Type: 'Common.ValueListParameterDisplayOnly', ValueListProperty: 'Country'},
+					{ $Type: 'Common.ValueListParameterDisplayOnly', ValueListProperty: 'StreetName'},
+					{ $Type: 'Common.ValueListParameterDisplayOnly', ValueListProperty: 'HouseNumber'},
+				]
+			}
+		}
+	);
+}
 
+////////////////////////////////////////////////////////////////////////////
```

Comment:
Should that section comment be before line 27? There, the UI annoations seem to begin IMHO

Reply:
The comment should have contrasted the other comment `Common` in line 12. As the block from 27 to 47 also added some Common annotations to elements of the entities I added it there.

The annotations starting in line 53 describe how the List / Object page UIs should look like, thats why I titled them with `UI`. Couldn't come up with a better name 😕